core/src/OpenMP/Kokkos_OpenMP_WorkGraphPolicy.hpp

```diff
@@ -74,7 +74,11 @@ class ParallelFor<FunctorType, Kokkos::WorkGraphPolicy<Traits...>,
 
  public:
   inline void execute() {
-#pragma omp parallel num_threads(OpenMP::impl_thread_pool_size())
+    // Work around NVHPC 22.5 ICE
+    int pool_size = OpenMP::impl_thread_pool_size();
```

Comment:
Why not `[[maybe_unused]]` instead of the workaround below?

Reply:
Sure, I'll change it. The PR started 6 months ago so we couldn't use C++17